src/java/org/sakaiproject/component/app/scheduler/jobs/ResetContentTypes.java

```diff
@@ -104,15 +105,15 @@ public void execute(JobExecutionContext arg0) throws JobExecutionException {
 		int numberOfSites = siteService.countSites(SiteService.SelectionType.ANY, null, null, null);
 		int first = 0;
 		int last = 0;
-		int pages = numberOfSites/100;
-		int remainder = numberOfSites%100;
+		int pages = numberOfSites/SITE_BATCH_SIZE;
+		int remainder = numberOfSites%SITE_BATCH_SIZE;
 
 		for(int x=0; x<=pages; x++) {
 			if(x == pages ) {
-				first = x*100;
+				first = x*SITE_BATCH_SIZE;
 				last = first+remainder;
 			} else {
-				first = x*100;
+				first = x*SITE_BATCH_SIZE;
 				last = first+99;
```

Comment:
first+SITE_BATCH_SIZE-1

Reply:
not the result of the calculation - replace the line 

last = first+99;

with:

last= first+SITE_BATCH_SIZE-1